README.md

```diff
@@ -32,13 +32,51 @@ conda env remove -n sleap-anipose
 ```
 
 ## Usage
+
+**From the command line**
+
+1. Activate the conda environment (assuming you installed via conda).
+```
+conda activate sleap-anipose
+```
+
+2. Generate a calibration board matching the details of the board presented before the cameras. 
+
+```
+slap-write_board --fname board.toml --board_width 8 --board_height 11 --square_length 24.0 marker_length --18.5 --marker_bits 4 dict_size 1000
```

Comment:
is this right? `marker_length --18.5`

Reply:
should have been --marker_length 18.5, fixed.